src/components/molecules/Progress/Progress.test.tsx

```diff
@@ -10,4 +11,13 @@ describe('<Progress />', () => {
     const results = await axe(container.innerHTML);
     expect(results).toHaveNoViolations();
   });
+  it('renders the component with the step text', () => {
+    const { queryAllByText } = render(<Progress totalSteps={5} currentStep={2} />);
+    expect(queryAllByText('Step 2 of 5')).toBeTruthy();
+    expect(queryAllByText('Step 2 of 5')).toHaveLength(2);
```

Comment:
why length 2 and without it length 1? 🤔

Reply:
I think this might be a jest/snapshot issue. In that case it detects a duplicate (some form of <span>).

Reply:
Found the culprit : `const results = await axe(container.innerHTML);` is not unmounting from the jest dom at the end of that step.

Reply:
I will write an issue separately to address this.